components/brave_rewards/core/endpoint/uphold/get_cards/get_cards.cc

```diff
@@ -32,7 +32,7 @@ mojom::Result GetCards::CheckStatusCode(int status_code) const {
     return mojom::Result::EXPIRED_TOKEN;
   }
 
-  if (status_code != net::HTTP_OK) {
+  if (status_code != net::HTTP_OK && status_code != net::HTTP_PARTIAL_CONTENT) {
```

Comment:
From what I gather, I think we'd be better off using the entire success range?

Reply:
Follow up issue created: https://github.com/brave/brave-browser/issues/35542